test/phoenix_live_view/integrations/live_view_test.exs

```diff
@@ -162,14 +162,12 @@ defmodule Phoenix.LiveView.LiveViewTest do
                "The temp is: 21[&quot;another&quot;, &quot;field&quot;]"
     end
 
-    @key_i 73
-    @key_d 68
     test "render_key|up|down", %{conn: conn} do
       {:ok, view, _} = live(conn, "/thermo")
       assert render(view) =~ "The temp is: 1"
-      assert render_keyup(view, :key, @key_i) =~ "The temp is: 2"
-      assert render_keydown(view, :key, @key_d) =~ "The temp is: 1"
-      assert render_keyup(view, :key, @key_d) =~ "The temp is: 0"
+      assert render_keyup(view, :key, %{"key" => "i"}) =~ "The temp is: 2"
```

Comment:
Huh I'm surprised these were passing 😱.

Reply:
I think they pass because the render functions accept whatever is passed to them and they are/were aligned. More rigorous tests would be useful here...

Reply:
Will take a look at the tests in another PR.